scripts/utils/environment.js

```diff
@@ -9,3 +12,76 @@ export const execOptions = { encoding: "utf8", stdio: "inherit" };
 export function configurePython(options = execOptions) {
   execSync("pipenv --python 3.8.17", options);
 }
+
+export function preExecuteChecks(pythonCheck = false, cookiecutterCheck = false) {
+  // Check if Node.js v18.16.0 is installed
+  try {
+    section("Checking Node version");
+    const nodeVersion = execSync("node --version", {
+      cwd: userdir,
+      shell: true,
+      encoding: "utf8"
+    }).trim();
+    if (!nodeVersion.includes("v18.16.0")) {
```

Comment:
Is there a reason why would we want to check for a specific version? I believe the version check should be a bit inclusive? at the very lease, for the minor versions. Maybe >= 18 or something. What do you think?

I have node 18.19.0 and I failed your check here.

```sh
aline@GU603HM:~/crowdbotics/modules$ npx . create --name esting --type django

> Checking Node version

> Node.js v18.16.0 is not installed. Found version: v18.19.0. Please install Node.js v18.16.0 before running this script.
```

Reply:
Additionally, I'd recommend maybe a utility that reads the version files so that the checks are more dynamic:
- .`nvmrc:` has the node version
- `.python-version`